src/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit.csproj

```diff
@@ -40,7 +42,38 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == '$(AndroidTargetFramework)'">
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\layout\TimeSlider.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\drawable\Thumb.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\drawable\TriangleFill.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\drawable\TriangleOutline.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\drawable\PauseFill.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\drawable\PauseOutline.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\drawable\PlayPauseButton.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\drawable\NextPreviousFill.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\drawable\NextPreviousOutline.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)Resources\drawable\NextPreviousButton.axml">
+      <SubType>Designer</SubType>
+    </AndroidResource>
```

Comment:
All these should be in `Android\Esri.ArcGISRuntime.Toolkit.Android.projitems`

Reply:
So, this is exceedingly weird... I can move the XML to the shared project, but the files themselves need to remain under the library project folder.  The build fails otherwise.